static/js/publisher/market/restrictions.js

```diff
@@ -40,7 +40,7 @@ const MEDIA_RESTRICTIONS = {
 };
 
 const ICON_RESTRICTIONS = {
-  accept: ["image/png", "image/jpeg", "image/svg"],
+  accept: ["image/png", "image/jpeg", "image/svg+xml"],
```

Comment:
good catch :+1:

Reply:
it was unintentional, because I was not able to upload svg :)